### ISSUE TITLE
sudo update-rc.d guvnor defaults is not working on my ubuntu machine

### ISSUE THREAD

**Issue by HERRKIN** [tableflip/guvnor#81]
I installed guvnor following the docs so I did this
```
sudo ln -s /usr/local/lib/node_modules/guvnor/scripts/init/sysv/guvnor /etc/init.d/guvnor
```
it worked, it created the symlink but when I tiped this:
```
sudo update-rc.d guvnor defaults
```
the term replied
```update-rc.d: /etc/init.d/guvnor: file does not exist```

**Comment by achingbrain** [comment]
Can you verify that the file `/usr/local/lib/node_modules/guvnor/scripts/init/sysv/guvnor` exists?  Node may have put it's global `node_modules` directory somewhere else on your system.

**Comment by HERRKIN** [comment]
no, it doesn't exist, it exists in usr/lib instead of usr/local/lib

**Comment by achingbrain** [comment]
I'm guessing you installed node via apt-get - if so, you want to create the symbolic link to the correct place:

```
sudo ln -s /usr/lib/node_modules/guvnor/scripts/init/sysv/guvnor /etc/init.d/guvnor
```

See the note about [apt-get](https://github.com/tableflip/guvnor#apt-get) in the README.
